final_task/rss_reader/news_converter.py

```diff
@@ -0,0 +1,18 @@
+import logging
+import json
+from dataclasses import asdict
+
+
+def news_as_json_str(feed_title, items_):
+    """ Convert news in json format
+
+    :type feed_title: str
+    :type items_: list of 'items.Item'
+    :rtype: str
+    """
+    logging.info('Converting items to dicts.')
+    map_of_dict_items = map(lambda item: asdict(item), items_)
```

Comment:
You can replace this with simple list comprehension, e.g. [asdict(item) for item in items_]

Reply:
fixed